catalogs/catalog-lakehouse-iceberg/src/main/java/com/datastrato/gravitino/catalog/lakehouse/iceberg/ops/IcebergTableOps.java

```diff
@@ -140,6 +148,32 @@ public void close() throws Exception {
     if (catalog instanceof AutoCloseable) {
       // JdbcCatalog need close.
       ((AutoCloseable) catalog).close();
+
+      // Does IcebergCatalog also use PG here?
+      closeMySQLCatalogResource();
+    }
+
+    // TODO(yuqi) add close for other catalog types such Hive catalog
+  }
+
+  private void closeMySQLCatalogResource() {
+    try {
+      // Close thread AbandonedConnectionCleanupThread
+      Class.forName(MYSQL_CATALOG_CONNECTION_CLEAN_UP_THREAD)
```

Comment:
suppose there are multi JDBC catalogs , is this thread shared by all drivers? If yes, there will be unexpected result.